### PR TITLE
[BEAM-2600] Add minimal python SDK harness container

### PR DESCRIPTION
**Pull request by herohde**
The container image does not support requirements files, main sessions, etc yet.

### CODE REVIEW

**Comment by herohde** [comment]
retest this please

**Comment by herohde** [comment]
retest this please

**Comment by herohde** [comment]
R: @aaltay 
CC: @kennknowles (for excluding the container for python presubmit)

**Comment by kennknowles** [comment]
Precommit alteration seems reasonable. TBH so far we don't have any non-leaf projects that also have a real build. I'm not sure there's any design reason for that, just like there is probably not a real design reason for having our pom inheritance match the directory structure.

File: sdks/python/container/boot.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import (
+"context"
+"flag"
+"fmt"
+"log"
+"os"
+"path/filepath"
+"strings"
+
+"github.com/apache/beam/sdks/go/pkg/beam/artifact"
```
[the remark this answers is not in the document]
Comment (herohde) [reply sdks/python/container/boot.go]: Go works with github out of the box, using an import path such as the above. Anyone can just run "go get github.com/apache/beam/sdks/go/..." to pull the code and use it.

The Go build in Maven does a bit of extra work to accommodate that path during development. See sdks/go/BUILD.md.

**Comment by herohde** [comment]
Thanks!

**Pull request closed without merging**